.travis.yml

```diff
@@ -2,8 +2,8 @@ sudo: false
 dist: trusty
 language: node_js
 node_js:
-  - 'node'
-  - 'lts/argon'
+  - '8'
```

Comment:
why not just `lts/*`? It'll make tests run faster too :)

Reply:
Because that's what the generator is spitting out in the current version. I can change it, though.